crates/accountsdb-rabbitmq/src/selectors.rs

```diff
@@ -0,0 +1,54 @@
+use std::collections::HashSet;
+
+use solana_program::instruction::CompiledInstruction;
+
+use crate::{interface::ReplicaAccountInfo, prelude::*};
+
+#[derive(Debug)]
+pub struct AccountSelector {
+    owners: HashSet<Box<[u8]>>,
+}
+
+impl AccountSelector {
+    pub fn from_config(owners: HashSet<String>) -> Result<Self> {
+        let owners = owners
+            .into_iter()
+            .map(|s| bs58::decode(s).into_vec().map(Vec::into_boxed_slice))
+            .collect::<Result<_, _>>()
+            .context("Failed to parse account owner keys")?;
+
+        Ok(Self { owners })
+    }
+
+    #[inline]
+    pub fn is_selected(&self, acct: &ReplicaAccountInfo) -> bool {
+        self.owners.is_empty() || self.owners.contains(acct.owner)
```

Comment:
I don't think this default is the right approach for us. Can we switch to filtering all of them out if the user specifies none. Bringing explicit support for all via config is a good future goal.